src/lib/components/app.svelte

```diff
@@ -3,25 +3,58 @@
     createSerialReadStore,
     type SerialStore,
   } from "$lib/stores/serialRead";
+  import { createBluetoothReadStore } from "$lib/stores/bluetoothRead";
   import Configurator from "./configurator.svelte";
   import ConnectionScreen from "./connectionScreen.svelte";
 
   let serialReadStore: SerialStore | null = null;
+  let bluetoothReadStore: SerialStore | null = null;
 
   const openPort = async (baudRate: number) => {
     const port = await navigator.serial.requestPort();
     serialReadStore = createSerialReadStore(port, baudRate);
     await serialReadStore.connect();
-    serialReadStore.subscribe((event) => {
-      if (event.eventType === "DISCONNECT") {
-        serialReadStore = null;
-      }
-    });
+    serialReadStore.subscribe(handleNotification);
+  };
+
+  const openBluetooth = async () => {
+    console.log("openBluetooth");
+
+    try {
+      const device = await (
+        navigator as Navigator & { bluetooth: any }
```

Comment:
I'd prefer to use [`@types/web-bluetooth`](https://www.npmjs.com/package/@types/web-bluetooth) instead of `any`s. Hopefully TypeScript gets built-in support for web Bluetooth but doesn't look like it's happened yet.